spec/services/builders/chartkick/defect_escape_rate_data_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe Builders::Chartkick::DefectEscapeRateData do
+  subject { described_class.send(:new, 1, '') }
```

Comment:
```suggestion
  subject { described_class.new(1, '') }
```
What about explicitly calling `new` ?